dusty/systems/docker/compose.py

```diff
@@ -47,6 +50,16 @@ def _compose_rm(compose_file_location, project_name, services):
         command += services
     check_and_log_output_and_error_demoted(command, env=get_docker_env())
 
+def _check_linked_container_status(client, container, assembled_specs):
+    app_or_service_name = _get_app_or_service_name_from_container(container)
+    linked_containers = links_for_app_or_service(app_or_service_name, assembled_specs)
+    for linked_name in linked_containers:
+        if _get_dusty_containers(client, [linked_name]) == []:
+            log_to_client('No running container for {0}, which is linked to by {1}.  Cannot restart {1}'.format(
```

Comment:
I wanted to log the specific linked container that is not running.  But I guess that can be returned by this function... it just might be a little messy